scripts/jenkins/cloud/ansible/roles/list_packages/tasks/list-packages.yml

```diff
@@ -29,12 +29,21 @@
     loop_var: item_virtual_hosts
   register: _result_packages_rpm
 
+# delegate_facts used to get hostvars from delegate hosts
 - name: Collect list of installed packages repo of origin from zypper
   shell: |
-    zypper --xmlout se -s -i -t package
+    if [ {{ hostvars[item_virtual_hosts].ansible_os_family }} = "Suse" ];then
```

Comment:
Is this really correct? Don't you just need `gather_facts: yes` and then `ansible_os_family` should be filled?

Reply:
nope because we delegate